src/presentational/AssignmentForm.jsx

```diff
@@ -0,0 +1,144 @@
+import { Component, PropTypes } from 'react';
+import { Link } from 'react-router';
+
+import InputElement from './InputElement';
+
+const initialState = {
+  classifications_target: '',
+  description: '',
+  duedate: '',
+  name: '',
+  students: [],
+  subjects: [],
+}
+
+
+class AssignmentForm extends Component {
+  constructor(props) {
+    super(props);
+    this.handleSubmit = this.handleSubmit.bind(this);
+    this.handleChange = this.handleChange.bind(this);
+    this.renderStudentList = this.renderStudentList.bind(this);
+    this.toggleStudent = this.toggleStudent.bind(this);
+    this.state = Object.assign({}, initialState);
+  }
+
+
+  toggleStudent(id) {
+    if (this.state.students.includes(id)) {
+      this.state.students = this.state.students.filter(student => student !== id)
+      this.setState({
+        students: this.state.students,
+      })
+    } else {
+      this.state.students.push(id);
+      this.setState({
+        students: this.state.students,
+      })
+    }
+  }
+
+  handleChange(e) {
+    let nextState = {};
+    nextState[e.target.name] = e.target.value;
+    this.setState(nextState);
+  }
+
+  handleSubmit(e) {
+    e.preventDefault();
+    const newAssignment = {};
+    for (const key in this.state) {
+      if (this.state.hasOwnProperty(key)) {
+        newAssignment[key] = this.state[key];
+      }
+    }
+    this.props.submitForm(newAssignment, this.props.params.classroomId);
+  }
+
+  renderStudentList(students) {
+    return (
+      <div>
+        {(students.length > 0) ?
+        <table className="table table-hover">
+          <thead>
+            <tr>
+              <th>Name</th>
+            </tr>
+          </thead>
+          <tbody>
+            {students.map((student) =>
+            <tr
+              className={this.state.students.find(id => id === student.id) ? 'success' : ''}
+              key={student.id}
+              onClick={this.toggleStudent.bind(this, student.id)}
```

Comment:
It looks like it can be clicked with a mouse, and has a selected/deselected state, but `tr` is not a focusable element, so it can't be toggled with the enter or space keys. `tr` doesn't have a `checked` attribute either, so it won't announce its selected state to the browser APIs used by VoiceOver and other assistive tech. You can write a bunch more JavaScript to add this behaviour, but you'd get it all for free if you used a checkbox to select each student.